packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -80,16 +76,13 @@ const liquidate = async (
   const runToBurn = AmountMath.min(proceeds.RUN, debt);
   // debt is fully settled, with runToBurn and shortfall
   assert(AmountMath.isEqual(debt, AmountMath.add(runToBurn, shortfall)));
-  trace('before burn', { debt, proceeds, overage, shortfall, runToBurn });
-  // TODO why grant this power; we can burn it from the caller
-  burnLosses(runToBurn, vaultZcfSeat);
```

Comment:
Now vault is updated with `liquidated` before the losses are burned, which updates the consumer of the vault. If `burnLosses` fails now the vault will still be "liquidated" to the user.

As I think this through I realize this could allow a state in which a vault's phase is `liquidated` and thus settled, but it's still in the vaultManager. Though that possibility true of master, albeit with one fewer case (failure of `burnLosses`) that could manifest it.